optuna/integration/allennlp.py

```diff
@@ -57,6 +57,11 @@ class AllenNLPExecutor(object):
     `config file <https://github.com/optuna/optuna/blob/master/
     examples/allennlp/classifier.jsonnet>`_.
 
+    .. note::
+        :class:`~optuna.integration.AllenNLPExecutor` also reads environment variables.
+        If a value is set in both a search space in Optuna and the environment variables,
+        the executor will use the value specified in the search space in Optuna.
```

Comment:
Thank you for moving the comment to the `AllenNLP.__init__`. The rendered result is as follows:

![image](https://user-images.githubusercontent.com/3255979/86085097-48300e80-bad9-11ea-8855-d3e5f631ca55.png)

How about clarifying another object (i.e., a search space of Optuna) of the `read` instead of omitting by `also`? If we add it, the note will be self-contained and easier to read.

The following is just an example:
`AllenNLPExecutor` reads environment variables as well as a search space of Optuna. If a value is set in both of them, the executor ...

Reply:
0c5a774 I tried updating my docstring so that it mentions AllenNLP.
How do you feel about the change?
If it doesn't make sense for you, I'll apply for your suggestion.
Thank you for the suggestion for improving English, it's really helpful for me as always.

Reply:
Thank you for your update. SGTM with a nit. Please take a look at https://github.com/optuna/optuna/pull/1449#pullrequestreview-440476843.